src/electionguard_cli/e2e_steps/submit_votes_step.py

```diff
@@ -0,0 +1,85 @@
+from typing import List
+import click
+from e2e_steps.e2e_step_base import E2eStepBase
+from electionguard_cli.cli_models import E2eInputs, BuildElectionResults
+from electionguard.data_store import DataStore
+from electionguard.ballot_box import BallotBox
+from electionguard.encrypt import EncryptionMediator
+from electionguard.election import CiphertextElectionContext
+from electionguard.manifest import InternalManifest
+from electionguard.utils import get_optional
+from electionguard.ballot import (
+    CiphertextBallot,
+    PlaintextBallot,
+)
+from electionguard_tools.factories.election_factory import (
+    ElectionFactory,
+)
+
+
+class SubmitVotesStep(E2eStepBase):
+    """Responsible for encrypting votes and storing them in a ballot store"""
+
+    def __encrypt_ballots(
+        self, plaintext_ballots: List[PlaintextBallot], encrypter: EncryptionMediator
+    ) -> List[CiphertextBallot]:
+        ciphertext_ballots: List[CiphertextBallot] = []
+        # Encrypt the Ballots
```

Comment:
This comment seem redundant to method. The recommendation would be to include this within method name.